app/scripts/modules/core/src/application/service/ApplicationReader.ts

```diff
@@ -41,6 +41,17 @@ export class ApplicationReader {
       });
   }
 
+  public static getApplicationPermissions(applicationName: string): IPromise<any> {
+    return API.one('applications', applicationName)
+      .withParams({
+        expand: false,
+      })
+      .get()
+      .then((fromServer: Application) => {
```

Comment:
Could we call this `app` or `application` so it is more clear what is coming from the server?

Reply:
done